docs/tech/django.md

```diff
@@ -0,0 +1,23 @@
+# Django
+
+We use the [Django][] Python framework heavily.
+
+You should use the latest stable release of Django *as of the start of the project*
+unless otherwise specified. Pin that version in your requirements so that the
+project won't break by accidentally being deployed on a newer version without
+testing.
+
+We strongly recommend you use our sample [django-skeleton][] as a starting point for
```

Comment:
The `django-skeleton` project is _very_ out-of-date and I'd prefer we didn't recommend it until/unless someone significantly updates it 😿